src/modules/app/app.module.ts

```diff
@@ -22,16 +22,16 @@ import {StartpageComponent} from "./components/startpage/startpage.component";
 import {ReactiveFormsModule} from "@angular/forms";
 import {MatInputModule} from "@angular/material/input";
 import {FlexLayoutModule} from "@angular/flex-layout";
-import { environment } from 'src/environments/environment';
+import {environment} from 'src/environments/environment.local';
```

Comment:
Die `environment.local` existiert nur lokal, darf hier nicht importiert werden. In `package.json` ab Zeile 4 stehen die scripts. `serve/build-local` greifen auf die `environment.local` zu. Der Code selbst muss immer auf `environment` zugreifen.

Reply:
Ah! Dann so? https://github.com/agri-gaia/DataDashboard/pull/18/commits/4dcac9fd71f057a2233d896538f7d72b48253f46